Tests/scripts/uninstall_packs_and_reset_bucket_cloud.sh

```diff
@@ -14,5 +14,5 @@ else
   gsutil -m cp -r "gs://$GCS_SOURCE_BUCKET/content" "gs://$GCS_MACHINES_BUCKET/$CLOUD_CHOSEN_MACHINE_ID/" > "$ARTIFACTS_FOLDER/Copy_prod_bucket_to_cloud_machine_cleanup.log" 2>&1
   echo "sleeping 120 seconds"
   sleep 120
-  python3 ./Tests/Marketplace/search_and_uninstall_pack.py --cloud_machine $CLOUD_CHOSEN_MACHINE_ID --cloud_servers_path $CLOUD_SERVERS_PATH --cloud_servers_api_keys "cloud_api_keys.json"
+  python3 ./Tests/Marketplace/search_and_uninstall_pack.py --cloud_machine $CLOUD_CHOSEN_MACHINE_ID --cloud_servers_path $CLOUD_SERVERS_PATH --cloud_servers_api_keys "cloud_api_keys.json" --unremovable_packs $UNREMOVABLE_PACKS
```

Comment:
```suggestion
  python3 ./Tests/Marketplace/search_and_uninstall_pack.py --cloud_machine $CLOUD_CHOSEN_MACHINE_ID --cloud_servers_path $CLOUD_SERVERS_PATH --cloud_servers_api_keys "cloud_api_keys.json" --unremovable_packs "${UNREMOVABLE_PACKS}"
```
Prevent globbing and expansion of bash variables